app/src/components/integrations/elements/integrationSettings/integrationSettings.jsx

```diff
@@ -157,6 +166,8 @@ export class IntegrationSettings extends Component {
               testConnection={this.testIntegrationConnection}
               onRemoveIntegration={this.removeIntegration}
               editAuthConfig={editAuthConfig}
+              integrationType={this.props.data.integrationType.name}
```

Comment:
This prop name is lil bit confusing. If you need `name` from `integrationType` please use relevant naming